src/Microsoft.Health.SqlServer/Features/Client/RetrySqlAction.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using EnsureThat;
+using Polly;
+
+namespace Microsoft.Health.SqlServer.Features.Client
+{
+    public class RetrySqlAction
```

Comment:
Do we need this class over taking a dependency on `ISqlServerTransientFaultRetryPolicyFactory` directly to create the policy?